src/Nethermind/Nethermind.Logging/PathUtils.cs

```diff
@@ -25,7 +28,17 @@ public static class PathUtils
     {
         public static string GetExecutingDirectory()
         {
-            return Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);;
+            var process = Process.GetCurrentProcess();
```

Comment:
I would recommend changing this to Property/readonly field. It won't change throughout the whole lifespan of the app. This way you don't have to cache it in variable where you use it, making a little less and more readable code.